### ISSUE TITLE
servo.detach() equivalent

### ISSUE THREAD

**Issue by jcmertz** [rwaldron/johnny-five#1722]
I'm struggling to figure out if there is a way to do something similar to the Arduino Servo libraries detach() function with johnny-five. I have a mechanism where servos are pulling open a spring loaded door, and then returning to their closed position. I am struggling to get the servos to line up nicely with the door in the closed position with the result being they are sometimes left under some load and end up running continuously to try to hold their position, which I don't need them to do. I'd like to be able to stop sending a PWM signal and tell them to "relax", but I can't seem to find a way to do that with the servo api in johnny-five. I have already tried stop(), but I think that just tells it to hold its current position. Anyone have any ideas?

**Comment by jcmertz** [comment]
Woops, realized there was already an issue for this [here](https://github.com/rwaldron/johnny-five/issues/1702). I'll close this and go add to that issue.
